docs/topics/proc_deploying-a-springboot-application-using-war-files.adoc

```diff
@@ -2,10 +2,13 @@
 = Deploying a {SpringBoot} application using WAR files
 // Consider placing this in the Spring Boot runtime documentation instead?
 // You can Package and deploy your application as a war file
+
+ifdef::spring-boot-1-5[]
```

Comment:
Just checking: Does this mean that WAR deployment is supported in 2.1?

Reply:
Yes, it  is supported in 2.1 only.